pipeline/train/train.sh

```diff
@@ -30,42 +35,90 @@ mkdir -p "${model_dir}/tmp"
 
 all_model_metrics=(chrf ce-mean-words bleu-detok)
 
-echo "### Training ${model_dir}"
+echo "### Preparing tsv datasets and config"
+
+# Generate a new OpusTrainer config based on a template to fill paths of the datasets
+new_config="${model_dir}/config.opustrainer.yml"
+cp "configs/opustrainer/${model_type}.yml" "${new_config}"
+
+# Iterate over the training sets
+# split the input string into an array
+IFS=',' read -ra datasets <<< "${train_set_prefixes}"
+IFS=',' read -ra alns <<< "${alignments}"
+# loop through the array and get both value and index
+for index in "${!datasets[@]}"; do
+    train_set_prefix="${datasets[index]}"
+    # OpusTrainer supports only tsv and gzip
+    # TODO: pigz is not found
+    tsv_dataset="${train_set_prefix}.${src}${trg}.tsv" #.gz"
+
+    if [ "${alignments}" != "None" ] ; then
+      train_aln="${alns[index]}"
+      # alignments are currently used only for student that uses one stage of training
+      echo "### Generating tsv dataset with alignments ${alignments}"
+      paste <(${COMPRESSION_CMD} -dc "${train_set_prefix}.${src}.${ARTIFACT_EXT}") \
+            <(${COMPRESSION_CMD} -dc "${train_set_prefix}.${trg}.${ARTIFACT_EXT}") \
+            <(${COMPRESSION_CMD} -dc "${train_aln}") \
+            >"${tsv_dataset}"
+#            | pigz -c >"${tsv_dataset}"
```

Comment:
If this is not needed, it's better to remove it.

Reply:
removed